.github/workflows/update-go-dependencies.yaml

```diff
@@ -0,0 +1,30 @@
+name: Update go dependencies and create PR
+
+on:
+  repository_dispatch:
+    types: update-dependencies
```

Comment:
Out of curiosity: What does this do?

Reply:
This allows you to trigger the action via curl, similar for a release, cf https://github.com/SAP/jenkins-library/blob/master/contrib/perform-release.sh

so if you replace "perform-release" with "update-dependencies" you can trigger the action on demand. I just added it in case someone wants to update the dependencies without waiting for monday, but did not really document it. Should this be documented?

Reply:
I've added a comment, hope that makes it more clear